### PR TITLE
feat: allow defining maxLength attribute for number type fields

### PR DESCRIPTION
**Pull request by anamulhaquemollah**

### CODE REVIEW

**Review by rikukissa — CHANGES_REQUESTED** [review]
@anamulhaquemollah correct me if I'm wrong but I think you don't need to change anything from core to fix this. 

Farajaland has a validation rule here that forces the value to be more than 0
https://github.com/opencrvs/opencrvs-farajaland/blob/develop/src/form/birth/optional-fields.ts#L89-L91

**Comment by anamulhaquemollah** [comment]
@rikukissa to limit data entry to two digits and get rid of error messages if user enters more than two digits. I thought to make  change in the core to make this happen.

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,7 @@
 - In advance search, any status tag is showing archived after search [#6678](https://github.com/opencrvs/opencrvs-core/issues/6678)
 - Fix first name issues when creating a user [#6631](https://github.com/opencrvs/opencrvs-core/issues/6631)
 - Show correct record option in certificate preview page when trying to print by RA [#6224](https://github.com/opencrvs/opencrvs-core/issues/6224)
+- Change condition of Number of previous births [#6356](https://github.com/opencrvs/opencrvs-core/issues/6356)
```
Comment (rikukissa) [inline CHANGELOG.md:29]: Use OpenCRVS core specific language here so for example, from Core's point of view this change would be 

"Allow defining `maxLength` attribute for number type fields"

**Review by rikukissa — COMMENTED** [review]
Few more small things and then this is ready to be merged in :) Good work

**Review by Zangetsu101 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,6 +34,8 @@
 - Fix first name issues when creating a user [#6631](https://github.com/opencrvs/opencrvs-core/issues/6631)
 - Show correct record option in certificate preview page when trying to print by RA [#6224](https://github.com/opencrvs/opencrvs-core/issues/6224)
 - Fix download failure for incomplete (without date of death) death declarations [#6807](https://github.com/opencrvs/opencrvs-core/issues/6807)
+- Allow defining maxLength attribute for number type fields [#6356](https://github.com/opencrvs/opencrvs-core/issues/6356)
+
```
Comment (Zangetsu101) [inline CHANGELOG.md:38]: @anamulhaquemollah Could we move this to the `New features` section please?

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (anamulhaquemollah) [reply CHANGELOG.md]: moved it to the New features section

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**